EIPS/eip-5496.md

```diff
@@ -1,10 +1,10 @@
 ---
 eip: 5496
 title: Multi-privilege Management NFT Extension
-description: Create sharable multi-privilege NFTs for EIP-721
+description: Create shareable multi-privilege NFTs for EIP-721
 author: Jeremy Z (@wnft)
 discussions-to: https://ethereum-magicians.org/t/eip-5496-multi-privilege-management-extension-for-erc-721/10427
-status: Draft
+status: Last Call
```

Comment:
You should rebase this PR against master.

Reply:
done